appsec/depaware.txt

```diff
@@ -0,0 +1,104 @@
+gopkg.in/DataDog/dd-trace-go.v1/appsec dependencies: (generated by github.com/tailscale/depaware)
+
+        gopkg.in/DataDog/dd-trace-go.v1/ddtrace                      from gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation+
+        gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext                  from gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation+
+        gopkg.in/DataDog/dd-trace-go.v1/internal/appsec              from gopkg.in/DataDog/dd-trace-go.v1/appsec
+        gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo        from gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation/httpsec+
+        gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation from gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation/httpsec+
+        gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation/grpcsec from gopkg.in/DataDog/dd-trace-go.v1/internal/appsec
+        gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation/httpsec from gopkg.in/DataDog/dd-trace-go.v1/appsec+
+     💣 gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/waf          from gopkg.in/DataDog/dd-trace-go.v1/internal/appsec
```

Comment:
AppSec is the bomb! 🤣

(Had to lookup that this means cgo or pkg unsafe in the [depaware README](https://github.com/tailscale/depaware))

Reply:
I added a note to `CONTRIBUTING.md` about what the bomb means so we have our own reference, too :)

On a related note, I tried out `depaware` on the C allocation profiling branch and it actually didn't flag `cmemprof`, even though the package definitely uses cgo. I've done a few experiments and I think the tool just doesn't flag cgo (only unsafe). I still think it would be useful to us.

Reply:
We are bomb indeed (and [we work at removing it](https://github.com/DataDog/dd-trace-go/pull/1278/files#diff-1035efe3d54a8e802494acf13d2f515927f9a61cfb66aa87076a9aebaae45742R223)) 😬

Also, I thought it would highlight the fact we only support darwin/amd64 and linux/amd64 but I guess it cannot due to our failsafe stubs which get compiled for any other target.

Reply:
> On a related note, I tried out depaware on the C allocation profiling branch and it actually didn't flag cmemprof, even though the package definitely uses cgo

@nsrip-dd Huh, that's annoying 😢. But yeah, still useful for sure.

@Julio-Guerra Just to be clear, I wasn't suggesting that AppSec is a nuclear weapon 🤣. I think AppSec is very good! I didn't intend to summon you from your well-deserved time off 🙈.

![image](https://user-images.githubusercontent.com/15000/171815497-c48cd4aa-381e-4654-837e-449035646d50.png)

That being said, the work on removing cgo is very exciting.

> Also, I thought it would highlight the fact we only support darwin/amd64 and linux/amd64 but I guess it cannot due to our failsafe stubs which get compiled for any other target.

If you look at the lines below this one in the same file, those packages get flagged as `LD` (linux/darwin). I think it just applies those flags to the package itself, not the packages including those packages?